source/content/core-updates.md

```diff
@@ -298,6 +298,15 @@ Whenever there's a new release of WordPress or Drupal core, updates will be avai
 
 <Partial file="drupal-8-8-warning.md" />
 
+## Supress WordPress Admin Notice
+
+By default WordPress admin will check for new upstream updates instead of the default WordPress update nag. You can disable this by setting the `DISABLE_PANTHEON_UPDATE_NOTICES` constant to `true` in your `wp-config.php` file. This only disabled the test and notice from the WordPress admin, you will still see updates in the Pantheon dashboard.
```

Comment:
Updates from WordPress core are also "upstream" updates (little-u), so maybe we should say "check for new Pantheon upstream updates". If you are concerned that people with a custom upstream might then be confused, maybe "check for new Pantheon or custom upstream updates, as applicable".

Reply:
Adjusted in https://github.com/pantheon-systems/documentation/pull/6060/commits/9c4951439c5bd5c8c578e4971e6a5ecfd26b5264